src/openforms/js/components/admin/form_design/registrations/camunda/FormFields.js

```diff
@@ -54,6 +47,121 @@ const getProcessSelectionChoices = (processDefinitions, processDefinition) => {
   return [processDefinitionChoices, versionChoices];
 };
 
+const ProcessDefinitionFields = ({
```

Comment:
Is there a reason why this component is part of FormFields.js?

Maybe nice to move this to a separate file?

Reply:
> Maybe nice to move this to a separate file?

I really, really want to do only the minimal work for Camunda since as far as I know nobody actually uses it and this should just be dropped from Open Forms core. If somebody needs it, we can offer it via an extension.